authentication/src/main/java/com/ecommercepractice/authentication/exception/EmailAlreadyUsedException.java

```diff
@@ -0,0 +1,21 @@
+package com.ecommercepractice.authentication.exception;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * The email that is attempting to save on the system
+ * is already in use by other Ath
+ */
+@Getter
+@Slf4j
```

Comment:
Do not use Slf4j in the exception clasess

Reply:
Ok, it's moved to exception handler.